src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -348,14 +380,15 @@ class GeometryToolComponentImpl extends BaseComponent<IProps, IState> {
   private handleCreatePoint = (point: JXG.Point) => {
 
     const handlePointerDown = (evt: any) => {
```

Comment:
It gets a little tricky keeping track of which `handlePointerDown` I'm reading...some kind of centralized state machine we talked about would be useful